src/components/directory-content.js

```diff
@@ -287,41 +300,39 @@ const DirectoryContent = () => {
         return (
             <div className={classes.cell}>
                 <Tooltip title={owner} placement="right">
-                    <Chip label={abbreviationFromUserName(owner)} />
+                    <Chip
+                        className={classes.chip}
+                        label={abbreviationFromUserName(owner)}
+                    />
                 </Tooltip>
             </div>
         );
     }
-
     function nameCellRender(cellData) {
         const elementUuid = cellData.rowData['elementUuid'];
         const elementName = cellData.rowData['elementName'];
         const objectType = cellData.rowData['type'];
         return (
             <div className={classes.cell}>
                 {!childrenMetadata[elementUuid] && (
-                    <CircularProgress size={25} />
+                    <CircularProgress
+                        size={18}
+                        className={classes.circularRoot}
+                    />
                 )}
                 {childrenMetadata[elementUuid] &&
                     objectType === elementType.STUDY && (
-                        <LibraryBooksOutlinedIcon />
+                        <LibraryBooksOutlinedIcon className={classes.icon} />
                     )}
-                <div
-                    style={{
-                        display: 'flex',
-                        alignItems: 'center',
-                        marginLeft: '10px',
-                    }}
-                >
-                    {childrenMetadata[elementUuid] ? (
-                        <div>{childrenMetadata[elementUuid].name}</div>
-                    ) : (
-                        <>
-                            {elementName}{' '}
-                            <FormattedMessage id="creationInProgress" />
-                        </>
-                    )}
-                </div>
+
+                {childrenMetadata[elementUuid] ? (
+                    <div>{childrenMetadata[elementUuid].name}</div>
+                ) : (
+                    <>
+                        {elementName}{' '}
```

Comment:
{' '} ?

Reply:
It adds a whitespace between `elementName` and the `creationInProgress` message